contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -60,12 +60,24 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
     uint256 public currentReserveImbalance;
     uint256 public totalMinted;
 
+    // tokenMultiplier allows this contract to scale the number of tokens it sends/receives to/from
+    // the source chain. This can be used to normalize the number of decimals places between
+    // the two subnets.
```

Comment:
```suggestion
    // tokenMultiplier allows this contract to scale the number of tokens it sends/receives to/from
    // the source chain. This can be used to normalize the number of decimals places between the tokens on
    // the two subnets.
```